compatibility.js

```diff
@@ -39,7 +39,7 @@ function removeEolFeatures (config, version) {
  * return {object} - The converted config object
  */
 const compatFunctions = {
-    v1: (config) => {
+    v1: (unmodifiedConfig, config) => {
```

Comment:
I'd consider swapping these, config is the primary functionality here and `unmodifedConfig` should be used less. This would allow v1, v2 to omit the argument too.